dop.sh

```diff
@@ -0,0 +1,18 @@
+## Install helm
+curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+chmod 700 get_helm.sh
+./get_helm.sh
+helm version
+
+## Creates secret
+kubectl create secret docker-registry directoronprem-registry-secret --docker-server=registry.mayadata.io --docker-username=${{ DOCKER_USERNAME }} --docker-password=${{ DOCKER_PASSWORD }}
```

Comment:
Indent this line of code & limit the line length to max 80 chars. We should not do horizontal scolling.